app/templates/empire/food.hbs

```diff
@@ -1,11 +1,21 @@
 <div>
   {{model.name}} currently has {{model.food}} food.
+  {{#if (gte model.foodProduction 0)}}
+    You generate {{model.foodProduction}} food per turn.
+  {{else}}
+    You are losing {{model.foodProduction}} food per turn.
+  {{/if}}
```

Comment:
I don't think it is possible anymore, at least with the current state of foodProduction.